packages/mobile/src/identity/privateHashing.ts

```diff
@@ -84,6 +89,11 @@ function* doFetchPhoneHashPrivate(e164Number: string) {
   return details
 }
 
+export function* balanceSufficientForQuotaRetrieval() {
+  const userBalance = yield select(stableTokenBalanceSelector)
+  return new BigNumber(userBalance).isGreaterThanOrEqualTo(ODIS_MINIMUM_DOLLAR_BALANCE)
```

Comment:
Please leave a TODO to check for CELO balance too. Once the signer update goes live, users with just CELO can query.